sh/gtf/gxa_references.conf

```diff
@@ -3,6 +3,7 @@ amborella_trichopoda ftp://ftp.ensemblgenomes.org/pub/plants/release-RELNO/gtf/a
 anas_platyrhynchos ftp://ftp.ensembl.org/pub/release-RELNO/gtf/anas_platyrhynchos_platyrhynchos/Anas_platyrhynchos_platyrhynchos.CAU_duck1.0.RELNO.gtf.gz
 anolis_carolinensis ftp://ftp.ensembl.org/pub/release-RELNO/gtf/anolis_carolinensis/Anolis_carolinensis.AnoCar2.0v2.RELNO.gtf.gz
 anopheles_gambiae ftp://ftp.ensemblgenomes.org/pub/metazoa/release-RELNO/gtf/anopheles_gambiae/Anopheles_gambiae.AgamP4.RELNO.gtf.gz
+apis_mellifera ftp://ftp.ensemblgenomes.org/pub/metazoa/release-54/gtf/apis_mellifera/Apis_mellifera.Amel_HAv3.1.RELNO.gtf.gz
```

Comment:
```suggestion
apis_mellifera ftp://ftp.ensemblgenomes.org/pub/metazoa/release-RELNO/gtf/apis_mellifera/Apis_mellifera.Amel_HAv3.1.RELNO.gtf.gz
```

Reply:
I will resolve as we both committed at same time